source/services/horizon.rst

```diff
@@ -92,3 +92,72 @@ Large Horizon table for django_session
      -rw-rw----  1 42434 42434 1.6K Dec  5 15:02 django_session.frm
      -rw-rw----  1 42434 42434 9.0M Dec  5 15:04 django_session.ibd
      ...
+
+Customize horizon theme
+=======================
+
+* Prepare overlay files
```

Comment:
It is unclear to me what the action in this step should be. From what follows, there is none. So can this be dropped?

Reply:
it should visible which files will touched